tests/fixtures/verify-cleanroom.test.ts

```
@@ -0,0 +1,894 @@
+        escapedPid = Number(await readFile(pidFile, 'utf8'));
+
+        expect(result.timedOut).toBe(true);
+        expect(Date.now() - startedAt).toBeLessThan(3_000);
+        expect(Number.isSafeInteger(escapedPid)).toBe(true);
+        expect(escapedPid).toBeGreaterThan(0);
+      } finally {
+        if (escapedPid > 0) {
+          try {
+            process.kill(escapedPid, 'SIGKILL');
+          } catch (error: any) {
+            if (error?.code !== 'ESRCH') throw error;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)